pkgs/applications/misc/wikicurses/default.nix

```diff
@@ -0,0 +1,22 @@
+{ stdenv, fetchurl, buildPythonApplication, python3, pythonPackages }:
+
```

Comment:
you can remove python3 and buildPythonApplication as they're attributes of pythonPackages